actions/FontConversion.php

```diff
@@ -188,23 +222,27 @@ protected function extractArchive($archiveFile)
      */
     protected function checkIntegrity($currentSelection, $oldSelection)
     {
-        $metadataExists = property_exists($currentSelection, 'metadata') && property_exists($currentSelection->metadata, 'name');
+        $metadataExists = property_exists($currentSelection, 'metadata')
+            && property_exists($currentSelection->metadata, 'name');
 
-        $prefExists = property_exists($currentSelection, 'preferences') && property_exists($currentSelection->preferences, 'fontPref')
-            && property_exists($currentSelection->preferences->fontPref, 'metadata') && property_exists($currentSelection->preferences->fontPref->metadata, 'fontFamily') ;
+        $prefExists = property_exists($currentSelection, 'preferences')
+            && property_exists($currentSelection->preferences, 'fontPref')
+            && property_exists($currentSelection->preferences->fontPref, 'metadata')
+            && property_exists($currentSelection->preferences->fontPref->metadata, 'fontFamily') ;
 
         if (
             ($metadataExists && $currentSelection->metadata->name !== 'tao')
             || ($prefExists && $currentSelection->preferences->fontPref->metadata->fontFamily !== 'tao')
             || (!$prefExists && !$metadataExists)
         ) {
-            return ['error' => __('You need to change the font name to "tao" in the icomoon preferences')];
+            return $this->error(__('You need to change the font name to "tao" in the icomoon preferences'));
         }
+
         $newSet = $this->dataToGlyphSet($currentSelection);
         $oldSet = $this->dataToGlyphSet($oldSelection);
 
-        return !!count(array_diff($oldSet, $newSet))
-            ? ['error' => __('Font incomplete!  Is the extension in sync width git?  Have you removed any glyphs?')]
+        return (bool)count(array_diff($oldSet, $newSet))
```

Comment:
We don't really need to count here, do we?
`return empty(rray_diff($oldSet, $newSet)) ?? $this->error(__('Font incomplete! Is the extension in sync width git? Have you removed any glyphs?'))` should do it.